frappe/social/doctype/energy_point_log/energy_point_log.py

```diff
@@ -300,6 +302,11 @@ def send_summary(timespan):
 
 	if not is_energy_point_enabled():
 		return
+
+	if not is_email_notifications_enabled() and\
```

Comment:
```suggestion
	if not is_email_notifications_enabled() or \
```

Alternatively,

Move `is_email_notifications_enabled()` check inside `is_email_notifications_enabled_for_type` function, so that way you do not have to import both functions.

eg.,
```py
def is_email_notifications_enabled_for_type():
	if not is_email_notifications_enabled():
		return False
	# remaining logic
```